install.sh

```diff
@@ -12,3 +12,7 @@ Rscript -e 'devtools::document()'
 
 # Install the package
 Rscript -e 'devtools::install()'
+
+# prevent terminal close
+printf "\n"
+read -p "Press enter to continue..."
```

Comment:
I'm not sure about this feature, could you elaborate why you've added it?
```suggestion
```

Reply:
This few lines is to prevent shell terminal exits immediately, even encounter errors. I think when I first run the shell, it just exited silently, and after awhile of trial & errors, just figure out i dont have yarn installed 🤣 . 
I think is better for developers to see the shell messages, is just my opinion, I can remove these lines if this is a bad idea